Bottom: 72655a880e952a055a6230f4b59db1a6011cfa21
Top:    8068b91b6e0211da0600fe0b8a80e030ad4de21e
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-18 13:24:55 -0700

Refresh of more-reset

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index ffe9b5e6ccf..fcf7f3ce70a 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -179,7 +179,7 @@ struct PythonFormat<
  * for such wrappers to just take a StringRef, because those may not be
  * null-terminated, so we'd wind up doing a full copy just to pass a
  * fixed string. Instead, wrappers can just take their arguments as
- * CStringArg, an callers can pass StringRefs, Twines, strings, or
+ * CStringArg, and callers can pass StringRefs, Twines, strings, or
  * const char*, and the right thing will happen. */
 class CStringArg {
   llvm::SmallString<32> storage;
